src/services/fileStorage/proxy-service.js

```diff
@@ -36,6 +38,38 @@ const sanitizeObj = (obj) => {
 	return obj;
 };
 
+const prepareThumbnailGeneration = (file, strategy, userId, data, props) => Promise.all([
+	strategy.getSignedUrl({
+		userId,
+		flatFileName: props.storageFileName,
+		localFileName: props.storageFileName,
+		download: true,
+		Expires: 3600 * 24,
+	}),
+	strategy.generateSignedUrl({
+		userId,
+		flatFileName: props.storageFileName.replace(
+			/(\..+)$/,
+			'-thumbnail.png',
+		),
+		fileType: data.type,
+	}),
+	Promise.resolve(file),
+]).then(([downloadUrl, signedS3Url, f]) => {
+	rp.post({
+		url: FILEPREVIEW_SERVICE_URI,
+		body: {
+			downloadUrl,
+			signedS3Url,
+			callbackUrl: `${FILE_PREVIEW_CALLBACK_URI}${f.thumbnailRequestToken}`,
```

Comment:
use path combine for resolve issues with /

Reply:
fixed